Makefile

```diff
@@ -28,22 +29,20 @@ all: build
 lint:
 	golangci-lint run -v
 
-.PHONY: build
-build: lint unit-test build-minimal generate-keys
-
-build-minimal: create-build-dir
-	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service cmd/main.go
-
-create-build-dir:
+$(BUILD_FOLDER):
 	mkdir -p $(BUILD_FOLDER)
 
 format:
 	goimports -w -l cmd/ internal/ subsystem/
 
+############
+# Generate #
+############
+
 generate:
 	go generate $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
 
-generate-from-swagger:
+generate-go-client:
```

Comment:
Why do you call it client? 
We generate service side as well.

Reply:
Wanted to be aligned with `generate-python-client`. Do you have a better name?

Reply:
you can still call it `generate-from-swagger` and split into `generate-go-client` `generate-go-service`?